style.css

```diff
@@ -361,9 +309,18 @@ ul li {
   align-items: center;
   padding: 12px;
   gap: 10px;
-  width: 166px;
-  height: 48px;
-  margin-left: -160px;
+  background: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.contactme button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
```

Comment:
- [ ] Kindly make sure as requested by the previous reviewer to align the `Get in touch` text correctly as the button looks in the provided Figma design. Adjust your width and padding in order to match the design.


    - ### YOUR DESIGN
    
    ![image](https://github.com/ntimanu/Portfolio/assets/87197412/a11471a9-f380-4a7e-b92e-dada447a1190)

    - ### FIGMA DESIGN
    
    ![image](https://github.com/ntimanu/Portfolio/assets/87197412/2fc2c974-2f03-429f-966a-4eeb91eb98cb)